README.md

```diff
@@ -1,2 +1,162 @@
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+# 📖 Math Magician <a name="about-project"></a>
+
+**Math Mystics** is a website for all fans of mathematics. It is a Single Page App (SPA) that allows users to:
+
+- Make simple calculations.
+- Read a random math-related quote.
+
+<!-- <div align="center">
+
+  <img src="./assets/home.png" alt="app-screenshot-1" width="auto"  height="auto" />
+  <img src="./assets/quote.png" alt="app-screenshot-2" width="auto"  height="auto" />
+  <img src="./assets/calculator.png" alt="app-screenshot-3" width="auto"  height="auto" />
+  </div> -->
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://reactjs.org/">React.js</a></li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+- Make simple calculations
+- Read a random math-related quote
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+[Live Demo Link](https://lucy-sees.github.io/MathMystics/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps:
+
+### Prerequisites
+
+In order to run this project you need:
+
+- Code Editor: VS Code
+- Web Browser
+- Node
+- Git
+
+### Setup
+
+- Clone the repository
+- Change directory into the project folder
+
+Example commands:
+
+```sh
+  git clone git@github.com:lucy-sees/MathMystics.git
+```
+
+### Install
+
+Install this project with:
+
+Example command:
+
+`npm install`
+
+### Usage
+
+To run the project, execute the following command:
+
+`npm start`
+
+### Run tests
+
+To run tests, run the following command:
+
+Example command:
+
+`npm test`
+
+ <p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **Lucy W. Mwangi**
+
+- GitHub: [@lucy-sees](https://github.com/lucy-sees)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- Add some tests
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
- [ ] [OPTIONAL] Hey @lucy-sees, the reason the issues page link is not working is because the relative path of  `../../issues/` is used it should normally work when the branch is merged but for your future reference you can use an absolute path like `https://github.com/lucy-sees/MathMystics/issues` to avoid such problems 😉.

Reply:
Thank you, I will make sure I update that